consul/consul_data_mappings.bal

```diff
@@ -95,15 +95,20 @@ function convertValues(json jsonStatus) returns Value {
 function convertToInt(json jsonVal) returns int {
     string stringVal = jsonVal.toString();
     if (stringVal != "") {
-        return check <int>stringVal;
+        var value = int.create(stringVal);
```

Comment:
`.create(...)` function is changed as `.convert(...)`
Please replace in all the places.